web/src/elements/forms/Form.ts

```diff
@@ -80,7 +80,7 @@ export function serializeForm<T extends KeyUnknown>(
         }
 
         if ("akControl" in inputElement.dataset) {
-            assignValue(element, inputElement.value, json);
+            assignValue(element, (inputElement as unknown as AkControlElement).json(), json);
```

Comment:
And with the taking control of `value`, this is how we ensure the values passed to authentik are correct.